RecoEgamma/EgammaIsolationAlgos/python/gamIsolationSequence_cff.py

```diff
@@ -8,4 +8,3 @@
     gamIsoDepositsTask,  
     gamIsoFromDepositsTask
 )
-gamIsolationSequence = cms.Sequence(gamIsolationTask)
```

Comment:
Please don't remove this one: it is still used in RecoEgamma/EgammaIsolationAlgos/python/egammaIsolationSequencePAT_cff.py